codejailservice/app.py

```diff
@@ -0,0 +1,87 @@
+import os
+import json
+import logging
+
+from codejail import jail_code
+
+from copy import deepcopy
+from flask import Flask, Response, jsonify, request
+
+LOG = logging.getLogger(__name__)
+
+app = Flask(__name__)
+env_config = os.getenv("FLASK_APP_SETTINGS", "codejailservice.config.ProductionConfig")
+app.config.from_object(env_config)
+
+def configure_codejail(app):
+    code_jail_settings = app.config["CODE_JAIL"]
+    python_bin = code_jail_settings.get('python_bin')
+    if python_bin:
+        user = code_jail_settings['user']
+        jail_code.configure("python", python_bin, user=user)
+    limits = code_jail_settings.get('limits', {})
+    for name, value in limits.items():
+        jail_code.set_limit(
+            limit_name=name,
+            value=value,
+        )
+    limit_overrides = code_jail_settings.get('limit_overrides', {})
+    for context, overrides in limit_overrides.items():
+        for name, value in overrides.items():
+            jail_code.override_limit(
+                limit_name=name,
+                value=value,
+                limit_overrides_context=context,
+            )
+
+configure_codejail(app)
+
+from codejail.safe_exec import SafeExecException, json_safe
+from codejail.safe_exec import not_safe_exec as codejail_not_safe_exec
+from codejail.safe_exec import safe_exec as codejail_safe_exec
+
+@app.route("/")
+def index():
+    return Response("Hello, world!", status=200)
+
+@app.route("/health")
+def health():
+    return Response("OK", status=200)
+
+@app.route("/api/v0/code-exec", methods=['POST'])
+def code_exec():
+    payload = json.loads(request.form["payload"])
```

Comment:
any reason to not use `request.get_json()` ?

Reply:
Yeah, since I am sending the payload as data in edx-platform https://github.com/edx/edx-platform/pull/27795/files#diff-71f6151337c687b676efeaf32541f1f02f8eb78cda5ad1c7553dbd1d8af22c87R65 the request.json object is None and the information is in the request.form. @MoisesGSalas Should I change the way the request is sent in edx-platform ? could you please check the edx-platofrm PR and give me some feedback ? thanks